src/components/dashboard/dashboardItem/ItemBase.js

```diff
@@ -45,6 +45,45 @@ const DashboardLink = ({ target, kind, children }) => {
     );
 };
 
+function AnalysisSubheader(props) {
+    const { analysis, date: formattedDate } = props;
+    const { t } = useTranslation(["dashboard", "apps"]);
+    const terminated = isTerminated(analysis);
+
+    const [runningTime, setRunningTime] = useState(null);
+
+    useEffect(() => {
+        const handleUpdateRunningTime = () => {
+            setRunningTime(formatDistanceToNow(new Date(analysis.start_date)));
```

Comment:
So I was talking to @slr71 about this feature the other day and she said this is the job submission time 🤔  and not the actual the time when job actually started running. She suggested we query job status history to get the actual time when the job started running.

Reply:
Sure, I'll take a look at that endpoint